src/platform_impl/windows/event_loop.rs

```diff
@@ -799,6 +799,19 @@ unsafe extern "system" fn public_window_callback<T>(
             commctrl::DefSubclassProc(window, msg, wparam, lparam)
         },
 
+        winuser::WM_NCLBUTTONDOWN => {
+            let mut runner = subclass_input.event_loop_runner.runner.borrow_mut();
+            if let Some(ref mut runner) = *runner {
+                let hwnd = runner.modal_redraw_window;
+                winuser::PostMessageW(hwnd, winuser::WM_NCLBUTTONDOWN, 0, 0);
+                if wparam == winuser::HTCAPTION as _ {
+                    winuser::PostMessageW(hwnd, winuser::WM_MOUSEMOVE, 0, 0);
+                }
+            }
+            drop(runner);
+            commctrl::DefSubclassProc(window, msg, wparam, lparam)
+        },
```

Comment:
Why do you forward the `WM_NCLBUTTONDOWN` message to the dummy window and trigger a redraw in there, instead of asking it to redraw directly from here?

Reply:
I checked again and also came to the conclusion that it's not necessary. I've pushed a new commit to remove the forwarding. I think it was leftover from sorting out the relationship with the dummy modal window.